lib/cocoapods/installer/xcode/multi_pods_project_generator.rb

```diff
@@ -11,34 +11,31 @@ class MultiPodsProjectGenerator < PodsProjectGenerator
         #
         def generate!
           container_project_path = sandbox.project_path
-          build_configurations = analysis_result.all_user_build_configurations
           all_platforms = aggregate_targets.map(&:platform)
-          object_version = aggregate_targets.map(&:user_project).compact.map { |p| p.object_version.to_i }.min
           # Generate container Pods.xcodeproj.
-          container_project_generator = ProjectGenerator.new(sandbox, container_project_path, [],
-                                                             build_configurations, all_platforms, object_version,
-                                                             config.podfile_path)
-          container_project = container_project_generator.generate!
+          container_project =
+            unless aggregate_targets.empty?
```

Comment:
why do we not generate this when the list of aggregate targets is empty? Does that break building a project with an empty podfile?